CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,31 @@
 
 import Foundation
 
-print("Hello, World!")
+func makeIcecream() {
+    for length in 1...8 {
+        for _ in 1...11
+        {
+            print("*", terminator: "")
+        }
+        print("")
+    }
+}
+
+func makeStick() {
+    for _ in 1...4 {
+        for width in 1...7 {
+            if width == 5 || width == 7 {
+                print("|", terminator: "")
+                
+            }
+            else {
+                print(" ", terminator: "")
+            }
+        }
+        print("")
```

Comment:
개행을 염두에 두고 쓴 코드 같아요.
그렇다면 아래처럼 수정해도 좋을 것 같아요.
```suggestion
        print()
```

Reply:
커밋을 보니 수정하셨군요! 👏
다음부터는 피드백에는 댓글이나 이모지로 반응을 남겨주시면 코드리뷰하는 입장에서 좀 더 편할 것 같습니다! ☺️